tests/python/rest_api/test_projects.py

```diff
@@ -461,11 +461,13 @@ def _test_import_project(self, username, project_id, format_name, data):
                 _content_type="multipart/form-data",
             )
             assert response.status == HTTPStatus.ACCEPTED
+            rq_id = json.loads(response.data).get("rq_id")
+            assert rq_id, "The rq_id was not found in the response"
 
             while True:
                 # TODO: It's better be refactored to a separate endpoint to get request status
                 (_, response) = api_client.projects_api.retrieve_dataset(
-                    project_id, action="import_status"
+                    project_id, action="import_status", rq_id=rq_id
```

Comment:
BTW, the change seems to be breaking. Why won't we just implement a new uniform API for file uploading and status checks? A protocol like yours, but more generic:

1. POST `/backup`, `/annotations`, `/dataset` -> 202 + upload_id
2. Optionally, CVAT-TUS protocol requests
3. GET `/status` to get request status

Surely, need to think more about this, it's just an idea.

Reply:
Need to think about it, but I'm not sure this approach should be implemented in this PR. I guess it will require a significant redesign.